media-libs/boca/boca-1.0~alpha20180913.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="C++ audio component framework and library"
+DESCRIPTION="BoCA is the component framework behind the fre:ac audio \
+converter. It provides unified interfaces for components like encoders, \
+decoders, taggers and extensions as well as code to support communication \
+between applications and their components."
+HOMEPAGE="https://github.com/enzo1982/BoCA/"
+COPYRIGHT="2007-2018 by Robert Kausch"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/enzo1982/BoCA/releases/download/v1.1-alpha-20180913/freac-cdk-1.1-alpha-20180913.tar.gz"
+CHECKSUM_SHA256="d8a6f59e3ccc68b3dbe5d77e5f3e97240691196467656714c678ccbf2932257b"
+SOURCE_DIR="freac-cdk-1.1-alpha-20180913"
+
+ARCHITECTURES="x86 x86_64"
```

Comment:
I did not check whether or not gcc2 can build boca, but I guess it does not. So we should have:
```diff
-ARCHITECTURES="x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```